src/utils/weatherUtils.js

```diff
@@ -0,0 +1 @@
+export const kelvinToCelsius = (kelvin) => kelvin - 273.15;
```

Comment:
OPTIONAL: By convention you can totally omit parentheses if there is only one parameter